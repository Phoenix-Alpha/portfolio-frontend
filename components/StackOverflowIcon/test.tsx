import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { StackOverflowIcon } from '.'

describe('<StackOverflowIcon />', () => {
  it('should renders correct <StackOverflowIcon />', () => {
    const wrapper = shallow(<StackOverflowIcon />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
