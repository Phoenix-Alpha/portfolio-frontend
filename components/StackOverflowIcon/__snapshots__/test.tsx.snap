// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StackOverflowIcon /> should renders correct <StackOverflowIcon /> 1`] = `
<svg
  height={20}
  viewBox="0 0 512 512"
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Stackoverflow
  </title>
  <path
    d="M392 440V320h40v160H64V320h40v120z"
  />
  <path
    d="M149.1 308.77l198.57 40.87 8.4-39.32-198.57-40.87zm26.27-93.12L359.22 300 376 263.76l-183.82-84.84zm50.95-89l156 127.78 25.74-30.52-156-127.78zM328 32l-33.39 23.8 120.82 160.37L448 192zM144 400h204v-40H144z"
  />
</svg>
`;
